src/sentry/static/sentry/app/components/events/user.jsx

```diff
@@ -36,7 +30,10 @@ var EventUser = React.createClass({
           title="User">
         <div className="user-widget">
           <div className="pull-left"><Gravatar email={user.email} size={84} /></div>
-          <DefinitionList data={children} />
+          <DefinitionList data={builtins} isContextData={false} />
+          {children &&
+            <DefinitionList data={children} isContextData={true} />
```

Comment:
I like this too since it prevents the entire set of values from being purely alphabetical and keeps "built in" values above user supplied stuff too.